codes.md

```diff
@@ -589,6 +594,26 @@ KATY/ALEX: Is this a sub-code of depiction?
 - Instances where participants drew something more like an Euler diagram, even though their description was more tree-like
   - [Session 009](https://osf.io/6n7pa/)
 
+### The "Describe your sketch" prompt sometimes yielded a recitation of the dataset (instead of a visual description of the paper)
+
+  - [Session 008](https://osf.io/wb4r5/) stated "I would probably explain it like the data set that you just gave me. There are 6 power plants (reads the data set for A and B)." Provided explanation of icons upon request.  
+  - [Session 014](https://osf.io/gzw56/): described (participant-selected) purpose and content for the dataset, but did not list data items or aspects of their drawing until they pointed to the table, saying "this is how I would represent the data."
+  - [Session 018](https://osf.io/ydnb9/): reads off all of the labels of the files
+  - [Session 027](https://osf.io/q9cn5/): reads file names (sentence structure very nearly matches the provided dataset)
+  - [Session 032](https://osf.io/bka6e/): did not describe the drawing or the data, simply said, "this is a representation of 6 power stations and each of the facilities they power... you can see what they all do in the legend." (The participant narrated extensively as they drew so it's possible they felt they didn't need to explain again)
+
+### The remaining 23 sessions included some sort of verbal description of what they drew (in response to the "Describe your sketch" prompt)
+
+  - Named a data abstraction or idiom: 9, 16, 17, 20, 23, 26, 28, 29
+
+    - NOTE: they did not always correctly name the abstraction or idiom that they used
+
+  - Only gave a visual description (usually of the icons or marks used): 6, 7, 10, 11, 12, 13, 15, 19, 21, 22, 24, 25, 30, 31, 33
+
+  - Did not describe the drawing (restated the dataset): 8, 14, 18, 27, 32
+
+  - This could have implications for accessibility -- when writing alt text for graphs or when presenting to visually impaired audiences.
+
```

Comment:
Could it be these two codes should be combined into one regarding how descriptions of drawings focused on data solely, marks solely, or a blend with the visual idiom -- is it purely the data, purely the drawing, or is it the data abstraction and essentially a verbal "key". (Is it that the ones who described the idioms often also described the marks?)

Reply:
Might be overcomplicating it by making it 2-D. Maybe just "individual data points --> individual icons --> relations of icons / positions of icons --> data abstraction"

Reply:
We decided to go with the single axis levels of details. Codes were combined.